app/screens/send-bitcoin-screen/payment-destination/lnurl.ts

```diff
@@ -0,0 +1,208 @@
+import { LazyQueryExecFunction } from "@apollo/client"
+import { Exact, UserDefaultWalletIdQuery, WalletCurrency } from "@app/graphql/generated"
+import { fetchLnurlPaymentParams } from "@galoymoney/client"
+import { LnurlPaymentDestination, PaymentType } from "@galoymoney/client/dist/parsing-v2"
+import crashlytics from "@react-native-firebase/crashlytics"
+import { getParams, LNURLPayParams } from "js-lnurl"
+import { createLnurlPaymentDetails } from "../payment-details"
+import {
+  CreatePaymentDetailParams,
+  DestinationDirection,
+  InvalidDestinationReason,
+  ResolvedLnurlPaymentDestination,
+  ReceiveDestination,
+  PaymentDestination,
+  ParseDestinationResult,
+} from "./index.types"
+import { resolveIntraledgerDestination } from "./intraledger"
+
+export type ResolveLnurlDestinationParams = {
+  parsedLnurlDestination: LnurlPaymentDestination
+  lnurlDomains: string[]
+  userDefaultWalletIdQuery: LazyQueryExecFunction<
+    UserDefaultWalletIdQuery,
+    Exact<{
+      username: string
+    }>
+  >
```

Comment:
```suggestion
  userDefaultWalletIdQuery: UserDefaultWalletIdLazyQueryHookResult
```

Reply:
there are many instance where the type that could be simplify with the diff above